docs/content/doc/features/gitea-directory.md

```diff
@@ -0,0 +1,53 @@
+---
+date: "2019-11-28:00:00+02:00"
+title: "The .gitea Directory"
+slug: "gitea-directory"
+weight: 40
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "features"
+    name: "The .gitea Directory"
+    weight: 50
+    identifier: "gitea-directory"
+---
+
+# The .gitea directory
+Gitea repositories can include a `.gitea` directory at their base which will store settings/configurations for certain features.
+
+## Templates
+Gitea v1.11.0 includes template repositories, and one feature implemented with them is auto-expansion of specific variables within your template files.  
+To tell Gitea which files to expand, you must include a `template` file inside the `.gitea` directory of the template repository.  
+Gitea uses [gobwas/glob](https://github.com/gobwas/glob) for its glob syntax. It closely resembles a traditional `.gitignore`, however there may be slight differences.
+
+### Example `.gitea/template` file  
+All paths should be from the base of the repository
+```gitignore
+# All .go files, anywhere in the repository
+**.go
```

Comment:
Will `FILE.GO` match this in Windows? Should it? Either way it's perhaps better to clarify this with an example here. e.g.:
```
# Batch files in both upper or lower case can be matched
**.[bB][aA][tT]
```

Reply:
You are correct, it does not match in windows.  
I can add the example.

Reply:
Done